test_fastdupes.py

```diff
@@ -0,0 +1,12 @@
+#execute with pytest
```

Comment:
It's a bit rude to just unilaterally decide that the tests will depend on pytest-specific features. (And, in fact, had you submitted this a year ago, I'd have asked you to rewrite it to be compatible with Nose.)

That said, the API presented is quite appealing and I *was* already considering a move to pytest so I'll